File: PRACTICA#3/GENERACIDAD/3/JAVA/Producto.java
public class Producto {
    private String nombre;
    private double precio;
    private String codigo;
    
    public Producto(String nombre, double precio, String codigo) {
        this.nombre = nombre;
        this.precio = precio;
        this.codigo = codigo;
    }
    
    public String getNombre() {
        return nombre;
    }
    
    public double getPrecio() {
        return precio;
    }
    
    public String getCodigo() {
        return codigo;
    }
    
    public String toString() {
        return "Producto: " + nombre + " - Precio: $" + precio + " - Código: " + codigo;
    }
}

File: PRACTICA#3/GENERACIDAD/3/PYTHON/Producto.py
class Producto:
    
    def __init__(self, nombre: str, precio: float, codigo: str):
        self.nombre = nombre
        self.precio = precio
        self.codigo = codigo
    
    def __str__(self) -> str:
        return f"Producto: {self.nombre} - Precio: ${self.precio} - Código: {self.codigo}"